chart/templates/trafficsplit-crd.yaml

```diff
@@ -0,0 +1,29 @@
+{{with .Values -}}
+---
+###
+### TrafficSplit CRD
+### Copied from https://github.com/deislabs/smi-sdk-go/blob/cea7e1e9372304bbb6c74a3f6ca788d9eaa9cc58/crds/split.yaml
+###
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
```

Comment:
Mind labelling this new CRD? We labelled all the global resources in #2971. Thanks!
```
  labels:
    {{.ControllerNamespaceLabel}}: {{.Namespace}}
```

Reply:
Is the controller namespace an appropriate thing to label on the CRD?  CRDs are cluster wide.  If there were multiple control planes installed, they would all share a single traffic split CRD.

Reply:
For now, we are disabling multi control plane setup. All the other global resources, including the service profile CRD, are labelled. The idea is to be able to find all Linkerd global and namespaced resources by using this label, for the purpose of determining if a control plane is installed. Previously, we rely only on checking for the existence of the `linkerd-config` config map, which is unreliable.